bin/npm-audit-wrapper

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+
+# This is just a stub binary for dev/debug purposes
```

Comment:
Please add a comment under here showing how you might invoke this from the command line.